site/en/docs/extensions/reference/runtime/index.md

```diff
@@ -1,6 +1,50 @@
 ---
 api: runtime
 ---
+The runtime API provides methods to support a number of areas:
+
+<dl>
+<dt>Message passing</dt>
+<dd><ul>
+<li>connect runtime.connect(extensionId: string, connectInfo: object): Port</li>
+<li>connectNative runtime.connectNative(application: string): Port</li>
+<li>sendMessage runtime.sendMessage(extensionId?: string, message: any, options: object, 
+responseCallback: function)</li>
+<li>sendNativeMessage runtime.sendNativeMessage(application: string, message: object, 
+responseCallback: function)</li>
```

Comment:
Should we link out here to the message passing article?  That seems like it would be a good standard for intros (linking to the more in-depth articles and / or how-tos)

Reply:
Done.